test/functional/test_framework/wallet.py

```diff
@@ -291,6 +290,8 @@ def create_self_transfer_multi(
                 height=0,
             ) for i in range(len(tx.vout))],
             "txid": txid,
+            "wtxid": tx.getwtxid(),
+            "vsize": tx.get_vsize(),
```

Comment:
I don't think it's necessary to add these fields since the callee can get them from the `CTransaction` object... though I don't feel that strongly about it.

Reply:
Done in https://github.com/bitcoin/bitcoin/commit/b21329d50639fc383d92c75ab7f20187de4ae0e9.